### PR TITLE
add being the CEO of the brave software

### PR DESCRIPTION
**Pull request by EsmaeilSaleh**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ This article helps you get started with JavaScript and furthers your understandi
 ## What is JavaScript?
 
 {{Glossary("JavaScript")}} is a powerful programming language that can add interactivity to a website.
-It was invented by Brendan Eich (co-founder of the Mozilla project, the Mozilla Foundation, and the Mozilla Corporation).
+It was invented by Brendan Eich (co-founder of the Mozilla project, the Mozilla Foundation, the Mozilla Corporation, and CEO of Brave Software.).
```
Comment (teoli2003) [inline files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:20]: Let's put it into a _timeless_ version: his bio is not really useful here. (People can google Brendan Eich if they want an up-to-date version).
```suggestion
It was invented by Brendan Eich.
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md (hunk 1 above)
Comment (teoli2003) [reply files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:20]: Exactly, it will be correct even in 10 years.

**Review by teoli2003 — APPROVED** [review]

**Comment by teoli2003** [comment]
Thanks a lot!

Looks good to me.

Congratulations upon your first merged commit here! Welcome aboard! :tada:

**Comment by EsmaeilSaleh** [comment]
Yes, Its more precise now. Thank you. My pleasure.

**Pull request merged**